HOST lewis.lbl.gov
only_i 27
only_code None
force_amber_prep True
dry_run False
... 27 2r8j
/net/chevy/raid1/nigel/amber/hi_lo
/net/chevy/raid1/nigel/amber/hi_lo/r8/2r8j
2r8j.pdb 4phenix_2r8j.pdb
local_archive None
archive PDB None
Reading from local mirror:
  /net/cci/pdb_mirror/pdb/r8/pdb2r8j.ent.gz
Model saved to /net/lewis/raid1/scratch1/nigel/amber/hi_lo/r8/2r8j/2r8j.pdb
Traceback (most recent call last):
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 199, in <module>
    run2(sys.argv[1:])
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 155, in run2
    files.append(get_pdb(id, data_type_, mirror, log, format=data_format))
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 213, in get_pdb
    data = fetch(id, data_type, mirror=mirror, format=format, log=log)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 150, in fetch
    data = libtbx.utils.urlopen(url)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/libtbx/utils.py", line 2069, in urlopen
    return urllib2.urlopen(*args, **kwds)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 404, in open
    response = self._open(req, data)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 422, in _open
    '_open', req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1184, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [Errno 110] Connection timed out>
local_archive None
archive PDB None
Reading from local mirror:
  /net/cci/pdb_mirror/pdb/r8/pdb2r8j.ent.gz
Model saved to /net/lewis/raid1/scratch1/nigel/amber/hi_lo/r8/2r8j/2r8j.pdb
Traceback (most recent call last):
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 199, in <module>
    run2(sys.argv[1:])
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 155, in run2
    files.append(get_pdb(id, data_type_, mirror, log, format=data_format))
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 213, in get_pdb
    data = fetch(id, data_type, mirror=mirror, format=format, log=log)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 150, in fetch
    data = libtbx.utils.urlopen(url)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/libtbx/utils.py", line 2069, in urlopen
    return urllib2.urlopen(*args, **kwds)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 404, in open
    response = self._open(req, data)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 422, in _open
    '_open', req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1184, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [Errno 110] Connection timed out>

==================================================
Summary of pdb4amber for: 2r8j.pdb
===================================================

----------Chains
The following (original) chains have been found:
A
B
P
Q
T
U

---------- Alternate Locations (Original Residues!))

The following residues had alternate locations:
None

---------- Mising heavy atom(s)

None
not prmtop_filename /net/lewis/scratch1/nigel/amber/hi_lo/r8/2r8j/4amber_2r8j.prmtop
not prmtop_filename /net/chevy/raid1/nigel/amber/hi_lo/r8/2r8j/4amber_2r8j.prmtop

	Running AmberPrep : 2r8j.pdb


==================================================
Running pdb4amber on 2r8j.pdb
==================================================

==================================================
Setting up library files for non-standard residues
==================================================

==================================================
Running elbow/antechamber for CPT 
==================================================

   Residue has metal at the centre of a coordination sphere. Using input
   geometry to generate restraints. Hydrogens may not be added.
   

MoleculeClass :  N: 2 Cl: 2 Pt: 1 (CHEMICAL COMPONENTS format)
	11 atoms
	10 bonds
	18 angles
	18 dihedrals
	0 rings (groups, rings, atoms) 0 0
	0 chirals

~> antechamber -i 4antechamber_CPT.pdb -fi pdb -o CPT.mol2 -fo mol2       -nc 0 -m 1 -s 2 -pf y -c bcc -at gaff2 -ek "qm_theory='AM1',grms_tol=0.0005,scfconv=1.d-10,maxcyc=0,ndiis_attempts=700,"

Sorry: Error: cannot run "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/amber/bin/sqm -O -i sqm.in -o sqm.out" of bcc() in charge.c properly, exit
